src/screens/Feed/pages/PostDetails.js

```diff
@@ -312,12 +323,39 @@ export default class PostDetails extends PureComponent {
         postLikesCount: isLiked ? postLikesCount - 1 : postLikesCount + 1,
       });
     }
+  }
+
+  deletePost = async () => {
+    try {
+      const { post } = this.state;
+      this.setState({ isDeleted: true });
```

Comment:
Doesn't look like `isDeleted` state is being used anywhere in this component.

Reply:
Good spot, thank you! I copied the functions over as-is.

Removed those now and added a TODO for a future issue because the user currently doesn't have any information as to whether the post was deleted or not. (Issue #431)

Reply:
Does deleting the post cause the render to break?

Reply:
No, nothing breaks. The Details screen just stays in place, showing the post, until you back out and refresh the feed/profile page (and the post is then gone, as expected).